packages/eslint-plugin-gestalt/src/__fixtures__/no-workflow-status-icon/invalid/match-properties.js

```diff
@@ -0,0 +1,7 @@
+import { Icon } from 'gestalt';
```

Comment:
make sure we are checking for renamed components too

See packages/eslint-plugin-gestalt/src/__fixtures__/no-spread-props/invalid-fixable-output.js

Reply:
Fixed! Current line 74